src/commands/init.ts

```diff
@@ -0,0 +1,80 @@
+import { Input } from "@oclif/parser";
+import * as fs from "fs-extra";
+import * as path from "path";
+import cli from "cli-ux";
+
+import TartCommand from "../TartCommand";
+
+export default class Init extends TartCommand {
+  static description = "Initializes tart for this project";
+
+  static flags = {
+    ...TartCommand.flags,
+  };
+
+  async init() {
+    // Skip
+  }
+
+  async run() {
+    const { flags, ...rest } = this.parse(
+      this.constructor as Input<typeof TartCommand.flags>
+    );
+
+    const configPath = flags.config as unknown as string;
+
+    const configPathExists = await fs.pathExists(configPath);
+
+    if (!configPathExists) {
+      if (await cli.confirm("A config file will be created, continue?")) {
+        const host = await cli.prompt("Please enter your database host");
+
+        const password = await cli.prompt(
+          "(Optional) Please enter your database password",
+          { required: false, type: "hide" }
+        );
+
+        const db = await cli.prompt(
+          "(Optional) Please enter your database name",
+          { required: false }
+        );
+
+        const user = await cli.prompt(
+          "(Optional) Please enter your database username",
+          { required: false }
+        );
+
+        await fs.outputJson(configPath, {
+          database: {
+            host,
+            password,
+            db,
+            user,
+          },
+          saveDir: ".tart",
+        });
+
+        this.log("The configuration file has been created successfully");
```

Comment:
maybe add the name of the file?

Reply:
https://www.npmjs.com/package/chalk

Reply:
Added the file name, we talked about adding chalk later (gonna resolve)